bluemira/codes/plasmod/equilibrium_2d_coupling.py

```diff
@@ -297,28 +306,30 @@ def solve_transport_fixed_boundary(
     figname = "Transport iteration "
     f, ax = None, None
 
-    for n_iter in range(max_iter):
-        plasma = create_plasma_xz_cross_section(
-            parameterisation,
-            transport_params,
-            paramet_params,
-            kappa_95,
-            delta_95,
-            lcfs_options,
-            f"from equilibrium iteration {n_iter}",
-        )
+    max_iter_inner = 20
```

Comment:
Fair, done! The range(50) thing is actually OK, because that ever fails there's something else going on. Ihaven't proven it, (though I think Emiliano Fable has) but that convergence loop is guaranteed within a very small number of iterations. Plus... 4 levels of convergence loop would be sad!

Reply:
Thanks, fair enough!